Firestore/core/test/firebase/firestore/testutil/CMakeLists.txt

```diff
@@ -12,17 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cc_library(
-  firebase_firestore_testutil_apple
-  SOURCES
-    app_testing.h
-    app_testing.mm
-  DEPENDS
-    FirebaseCore
-    absl_strings
```

Comment:
The bug here was a missing EXCLUDE_FROM_ALL which would do the equivalent and be consistent with the way we've done it elsewhere

https://github.com/firebase/firebase-ios-sdk/blob/94f92d41409106587d40575cec8d6e24a62de4c2/Firestore/core/src/firebase/firestore/util/CMakeLists.txt#L53

Reply:
Done.